target/linux/ramips/dts/mt7620a_netcore_nw5212.dts

```diff
@@ -0,0 +1,143 @@
+#include "mt7620a.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "netcore,nw5212", "ralink,mt7620a-soc";
+	model = "Netcore NW5212";
+
+	aliases {
+		led-boot = &led_wlan2g;
+		led-failsafe = &led_wlan2g;
+		led-upgrade = &led_wlan2g;
```

Comment:
Using wlan2g is not a good idea here, since led-* will be evaluated after the default-trigger and thus probably remove its function.

Reply:
should we just remove these lines?

Reply:
Then you won't have any notification about the boot state etc. Maybe just hijack one of the other leds that don't have a DT trigger.